src/components/Customer.js

```diff
@@ -33,85 +43,153 @@ export class Customer extends Component {
-              <tr key={cus.id}>
-                <td>{cus.id}</td>
-                <td>{cus.name}</td>
-                <td>{cus.age}</td>
-                <td>{cus.address}</td>
-                <td>
-                  <ButtonToolbar>
-                    <Button
-                      className="mr-2"
-                      variant="info"
-                      onClick={() =>
-                        this.setState({
-                          editModalShow: true,
-                          cusid: cus.id,
-                          cusname: cus.name,
-                          cusage: cus.age,
-                          cusaddress: cus.address
-                        })
-                      }
-                    >
-                      Edit
-                    </Button>
-                    <Button
-                      className="mr-2"
-                      variant="danger"
-                      onClick={() => this.deletecust(cus.id)}
-                    >
-                      Delete
-                    </Button>
-                    <EditCustModal
-                      show={this.state.editModalShow}
-                      onHide={editModalClose}
-                      cusid={cusid}
-                      cusname={cusname}
-                      cusage={cusage}
-                      cusaddress={cusaddress}
-                    />
-                    <EditCustModal
-                      show={this.state.editModalShow}
-                      onHide={editModalClose}
-                      cusid={cusid}
-                      cusname={cusname}
-                      cusage={cusage}
-                      cusaddress={cusaddress}
-                    />
-                  </ButtonToolbar>
-                </td>
+      <Router>
+        <div>
+          <Table className="mt-4" striped bordered hover size="sm">
+            <thead>
+              <tr>
+                <th>CustomerID</th>
+                <th>CustomerName</th>
+                <th>AGE</th>
+                <th>ADDRESS</th>
+                <th>Options</th>
               </tr>
-            ))}
-          </tbody>
-        </Table>
-        <ButtonToolbar>
-          <Button
-            variant="primary"
-            onClick={() => this.setState({ addModalShow: true })}
-          >
-            Add Customer
-          </Button>
-          <AddCustModal show={this.state.addModalShow} onHide={addModalClose} />
-        </ButtonToolbar>
-      </div>
+            </thead>
+            <tbody>
+              {cust.map(cus => (
+                <tr key={cus.id}>
+                  <td>{cus.id}</td>
+                  <td>{cus.name}</td>
+                  <td>{cus.age}</td>
+                  <td>{cus.address}</td>
+                  <td>
+                    <ButtonToolbar>
+                      <Button
+                        className="mr-2"
+                        variant="info"
+                        onClick={() =>
+                          this.setState({
```

Comment:
It is often better to define the function separately and set onClick to a reference. 
i.e `onClick={this.editCustomer}` and `editCustomer = (customer) => () => this.setState({...})` as a function in the class